app.js

```diff
@@ -2,3 +2,58 @@
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
 //  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
+
+var express = require('express');
+var app = express();
+var foodTrucks = require('./trucks');
+
+app.use(express.static('public'));
+
+app.get('/', function (request, response) {
+    response.redirect('/foodTrucks');
+});
+
+app.get('/trucks/:name', function (request, response) {
+    var truck = foodTrucks.getTruck(request.params.name);
+    var list;
+    
+    if (truck.name) {
+        list += "<li>Name: " + truck.name;
```

Comment:
Browsers are forgiving with markup, but you should add the closing li tag when constructing your markup.